bundles/EcommerceFrameworkBundle/FilterService/FilterType/ElasticSearch/MultiSelect.php

```diff
@@ -15,16 +15,20 @@
 namespace Pimcore\Bundle\EcommerceFrameworkBundle\FilterService\FilterType\ElasticSearch;
 
 use Pimcore\Bundle\EcommerceFrameworkBundle\FilterService\FilterType\AbstractFilterType;
+use Pimcore\Bundle\EcommerceFrameworkBundle\IndexService\Config\AbstractConfig;
 use Pimcore\Bundle\EcommerceFrameworkBundle\IndexService\ProductList\ProductListInterface;
+use Pimcore\Bundle\EcommerceFrameworkBundle\IndexService\Worker\ElasticSearch\AbstractElasticSearch;
 use Pimcore\Bundle\EcommerceFrameworkBundle\Model\AbstractFilterDefinitionType;
 use Pimcore\Model\DataObject\Fieldcollection\Data\FilterMultiSelect;
 
 class MultiSelect extends \Pimcore\Bundle\EcommerceFrameworkBundle\FilterService\FilterType\MultiSelect
 {
     public function prepareGroupByValues(AbstractFilterDefinitionType $filterDefinition, ProductListInterface $productList)
     {
-        $field = $this->getField($filterDefinition);
-        $productList->prepareGroupByValues($field, true, !$filterDefinition->getUseAndCondition());
+        if (method_exists($filterDefinition, 'getUseAndCondition')) {
```

Comment:
we should check `$filterDefinition instanceof \Pimcore\Model\DataObject\Fieldcollection\Data\FilterMultiSelect`, other wise throw `InvalidConfigException`